pkg/sensors/tracing/kprobe_test.go

```diff
@@ -6142,6 +6142,60 @@ spec:
 	err = jsonchecker.JsonTestCheck(t, checker)
 	assert.NoError(t, err)
 }
+func TestKprobeUserStackTrace(t *testing.T) {
```

Comment:
so you already added the test in your first commit and then you modified it in your "add the test commit", could you cleanup this and properly add this new test in a separate one?